qualtran/bloqs/factoring/rsa/__init__.py

```diff
@@ -0,0 +1,34 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+# isort:skip_file
+
+r"""Bloqs for breaking RSA cryptography systems via integer factorization.
+
+RSA cryptography is a form of public key cryptography based on the difficulty of
+factoring the product of two large prime numbers.
+
+Using RSA, the cryptographic scheme chooses two large prime numbers p, q, the product
+N, λ(n) = lcm(p - 1, q - 1), an integer e such that 1 < e < λ(n),
```

Comment:
why is lambda written as a function of n when the dependency doesn't show up in its definition? You don't actually define little n anywhere